source/docs/articles/sites/domains.md

```diff
@@ -1,52 +1,97 @@
 ---
-title: Domains and SSL Tool
+title: Domains and SSL
 description: Detailed information on adding a domain to your Pantheon Drupal or WordPress site.
 category:
   - developing
   - managing
   - going-live
 ---
+
 Pantheon provides distinct base domains for the consistent access of the Development, Testing, and Live production environments for each site on the platform from the moment they are launched.
 
 All sites on Pantheon can be divided into three base domain URL patterns.
 
 **Note:** In the example base domain patterns, `env` stands for either `dev`, `test`, or `live`; and `sitename` is the chosen name for the site:
 
-* [Pantheon.io base domains](https://pantheon.io/docs/articles/sites/domains/using-pantheon-io-for-better-uptime/): `env-sitename.pantheon.io`
+* [Pantheon.io base domains](https://pantheon.io/docs/articles/sites/domains/using-pantheon-io-for-better-uptime/) for sites created after October 22, 2014 and without a custom base domain: `env-sitename.pantheon.io`
 * Legacy GotPantheon.com base domains: `env.sitename.gotpantheon.com` or `env-sitename.gotpantheon.com`
 * [Custom organizational base domains](https://pantheon.io/docs/articles/organizations/base-domains/): `env-sitename.custom.domain.tld`
 
-Once sites are launched at a paid service level, custom domains can be added to each environment. Securing visitor traffic with HTTPS communication requires uploading a custom certificate and setting the DNS `A` record to a dedicated IP. For all sites in all configurations, the recommended DNS records will be provided on the Domains and SSL tab.
+Once site plans are changed to a paid service level, custom domains can be added to each environment. Securing visitor traffic with HTTPS communication requires uploading a custom certificate and setting the DNS `A` record to a dedicated IP. For all sites in all configurations, the recommended DNS records will be provided on the Domains and SSL tab.
 
-* [Adding a Domain to a Site Environment](/docs/articles/sites/domains/adding-a-domain-to-a-site-environment/)
-* [Enable SSL for Secure HTTPS Communication](/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication/)
-* [DNS Records for Directing Your Domain to Your Pantheon Site](/docs/articles/sites/domains/dns-records-for-directing-your-domain-to-your-pantheon-site/)
+## Step 1: Decide Which Type of Domain You Want
+
+**Examples of HTTPS Sites**
+
+https://www.example.com
```

Comment:
- [ ] Highlight as Recommended, 
- [ ] link to why 
- [ ] and/or possibly briefly explain.